producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
+
+This extension provides support for sending Change Data Capture(CDC) events to an Apache Kafka®.
+It is designed to work with the DataStax Stargate.
+
+== Kafka CDC - Stargate integration
+
+This extension is designed to integrate with Stargate. It uses a link:https://www.osgi.org/[OSGi] framework for registering the extension within the Stargate.
```

Comment:
Also, if you want to be specific you could link to Apache Felix which is the OSGI lib we use.
```suggestion
This extension is designed to integrate with Stargate. It uses an link:https://www.osgi.org/[OSGi] framework for registering the extension within Stargate.
```

Reply:
re: adding felix, I am not sure if we want to be such specific.